src/NuGet.Clients/VisualStudio/Extensibility/IVsFrameworkParser.cs

```diff
@@ -21,5 +21,18 @@ public interface IVsFrameworkParser
         /// <exception cref="ArgumentException">If the provided string cannot be parsed.</exception>
         /// <returns>The parsed framework.</returns>
         FrameworkName ParseFrameworkName(string shortOrFullName);
+
+        /// <summary>
+        /// Gets the shortened version of the framework name from a <see cref="FrameworkName"/>
```

Comment:
The remarks pretty much explain it all here, but you I think it's worth calling out that the short name is NuGet's own notation for a System.FrameworkName. It was designed to be used for folder names.